### PR TITLE
Fix Text Alignment on WelcomeScreen

### PR DESCRIPTION
**Pull request by qchau97**
Fix Text Alignment on WelcomeScreen

### CODE REVIEW

File: lib/src/screens/welcome_screen.dart [hunk 1]
```diff
@@ -35,21 +35,31 @@ class WelcomeScreen extends StatelessWidget {
...
-                          height: 1.5,
+                      Padding(
+                        padding: const EdgeInsets.symmetric(
+                          horizontal: 25,
+                        ),
+                        child: Text(
+                          Welcome.description,
+                          textAlign: TextAlign.left,
+                          style: AppStyles.kTextRegularSmoky16.copyWith(
+                            height: 1.5,
```
Comment (huyntcua) [inline lib/src/screens/welcome_screen.dart:61]: It's not a big deal but why do you know height is 1.5 ?

**Review by tonysodeep — APPROVED** [review]
em thấy ok rồi ấy chị

**Review by huyntcua — APPROVED** [review]
Great!

File: lib/src/screens/welcome_screen.dart (hunk 1 above)
Comment (qchau97) [reply lib/src/screens/welcome_screen.dart:61]: @huyntcua height = fontSize * x => x = height/fontSize = 24/16 = 1.5 ^^

image

Comment (huyntcua) [reply lib/src/screens/welcome_screen.dart:61]: Nice!

**Pull request merged**